Sources/SFSafeSymbols/Symbols/SFSymbol+AllSymbols+1.0.swift

```diff
@@ -1647,7 +1647,7 @@ extension SFSymbol {
             zCircleFill,
             zSquare,
             zSquareFill,
-            zzz
+            zzz,
```

Comment:
Ah, and one other little detail: I would suggest not to put a comma behind the last entry of the set